CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** [Open a Q&A Discussion]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Idea Discussion]
+- **💻 I want to code:** See below
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
+including sensitive information to the issue tracker, or elsewhere in public.
+Instead sensitive bugs must be sent by email to spenserblack01@gmail.com.
+
+## Contributing code

```

Comment:
Somewhere in this section we might want to suggest using (or at least complying with) the EditorConfig file.